package.json

```diff
@@ -1,47 +1,48 @@
 {
```

Comment:
I'm puzzled why this entire file is shown as updated when it seems only `shared` has now been added in `workspaces`.